Networking/Networking/Model/WCPayAccountStatusEnum.swift

```diff
@@ -0,0 +1,100 @@
+import Foundation
+
+/// Represents all of the possible Site Plugin Statuses in enum form
+///
+public enum WCPayAccountStatusEnum: Decodable, Hashable, GeneratedFakeable {
+    /// Complete: This is the normal state for a fully functioning WCPay account. The merchant should be able to collect
+    /// card present payments.
+    case complete
+
+    /// restricted: This state occurs when there is required business information missing from the account.
+    /// If hasOverdueRequirements is also true, then the deadline for providing that information HAS PASSED.
+    /// The merchant will probably NOT be able to collect card present payments.
+    /// Otherwise, if hasPendingRequirements is true, then the deadline for providing that information has not yet passed.
+    /// The deadline is available in currentDeadline. The merchant will probably be able to collect card present payments
+    /// until the deadline.
+    /// Otherwise, if neither hasOverdueRequirements nor hasPendingRequirements is true, then the account is under
```

Comment:
If you like, you can also use backticks to identify the references to code. 

```suggestion
    /// Otherwise, if neither `hasOverdueRequirements` nor `hasPendingRequirements` is true, then the account is under
```

Helps with reading I think. 

![image](https://user-images.githubusercontent.com/198826/113335234-e3ad1480-92e1-11eb-9385-e254ac7a992b.png)

Reply:
Fixed in https://github.com/woocommerce/woocommerce-ios/pull/3881/commits/b14c02d7dea186086d7c96b87bccd3f76ea5eefd